repo/blob/s3/s3_storage_test.go

```diff
@@ -51,6 +54,7 @@ const (
 	testBucketEnv          = "KOPIA_S3_TEST_BUCKET"
 	testLockedBucketEnv    = "KOPIA_S3_TEST_LOCKED_BUCKET"
 	testRegionEnv          = "KOPIA_S3_TEST_REGION"
+	testRoleEnv            = "KOPIA_S3_TEST_ROLE"
```

Comment:
@julio-lopez  - Since I've introduced a new env var here, do I have to update some other config file or test package to ensure that a particular role is used for the new test ?  I searched KOPIA_S3_TEST_REGION and found that there is a reference to it in provider-tests.yml 

```
KOPIA_S3_TEST_REGION: ${{ secrets.KOPIA_S3_TEST_REGION }}
```

Should I add a line for KOPIA_S3_TEST_ROLE here too ? Where does `secrets` come from ? Is this a secret setup somewhere in a github account ?